reddit_io.py

```diff
@@ -52,6 +52,9 @@ class RedditIO(threading.Thread, LogicMixin):
 			'truncate': '<|eo',
 	}
 
+	_subreddit = None
```

Comment:
I'm thinking that these are redundant because the default is being set later if the config value doesn't exist.
Let's move the default values into this variable declaration (ie so _subreddit = 'test') and same for the new_submission frequency

Reply:
Done.